### PR TITLE
restyled .ORG to .MD

### PR DESCRIPTION
**Pull request by amiruqdah**
I did a few stylistic changes to your .ORG files (including your README) to make it more easier for new comers to navigate to other instructional files. .MD is more widely used than ORG Mode and is much more easier to edit and write with!

I'll be contributing to the source later on ;)

### CODE REVIEW

**Comment by hhirsch** [comment]
Thank you for the PR.

File: CODING_GUIDELINES.md [hunk 1]
```diff
@@ -1,31 +1,32 @@
...
+- Encoding conversion macros (```L"", _T(""), T(""), …```) are considered a plague and will be rejected 
 (unless they are in platform-dependent source files which only gets compiled on those platforms).
-*** Preprocessor
-**** Include guards
-Include guards have an uppercase name reflecting the current header file name, and are terminated by ~_HPP~.
 
-They are not to be prefixed or suffixed by any underscore.
+### Preprocessor
+#### Include guards
+**Include guards** have an uppercase name reflecting the current header file name, and are terminated by ~_HPP~.
```
Comment (SGOrava) [inline CODING_GUIDELINES.md]: _HPP should be highlighted

File: CODING_GUIDELINES.md [hunk 2]
```diff
@@ -1,31 +1,32 @@
...
+
+*They are not to be prefixed or suffixed by any underscore.*
 
 In fact, all global names (that includes defines) starting with one underscore and a capital letter or two underscores 
 are reserved by the C and C++ standard. 
 As of the 2012-01-16 C++11 Working Draft, this is specified in 17.6.4.3.2/1 ~[global.names]~.
```
Comment (SGOrava) [inline CODING_GUIDELINES.md]: [global.names] should be highlighted

**Comment by amiruqdah** [comment]
@hhirsch @SGOrava Thank you for merging and correcting what I missed. Who removed the cake icon? :cry:

**Comment by hhirsch** [comment]
@amiruqdah I did.

**Comment by amiruqdah** [comment]
:sob: :sob: haha, alright.

**Pull request merged**